tests/test_sseclient.py

```diff
@@ -30,8 +30,8 @@ def __init__(self, payload, recorder):
     def send(self, request, **kwargs):
         resp = super(MockSSEClientAdapter, self).send(request, **kwargs)
         resp.url = request.url
-        resp.status_code = self._status
-        resp.raw = six.BytesIO(self._data.encode())
+        resp.status_code = self._statuses[0]
```

Comment:
See if we can do without this. Check my comment below.

Reply:
Done. I still have to change `self._status` to `self.status`, and `self._data` to `self.data`.